TemplateBicep/main.bicep

```diff
@@ -39,6 +39,9 @@ param gitUsername string = 'Azure'
 @description('The Git version to use. Default is 2.2.0.')
 param version string = '2.2.0'
 
+@description('The location of the cli tool to be used for device provisioning.')
+param loraCliUrl string = 'https://github.com/Azure/iotedge-lorawan-starterkit/releases/download/v${version}/lora-cli.linux-musl-x64.tar.gz'
```

Comment:
Why defaulting to musl? Any specific reason?

Reply:
We use the tool within the [az-cli docker image](https://github.com/Azure/azure-cli/blob/dev/Dockerfile). It is built on top of Alpine. Running "regular" Linux was throwing errors due to missing libraries.

Reply:
Makes sense, thanks for the context